tests/plugins/api/test_default_definition_body_plugin.py

```diff
@@ -0,0 +1,51 @@
+from unittest import TestCase
+from mock import Mock, patch
```

Comment:
You could optionally fix this import order here- keep them in alphabetical order.